### PR TITLE
dnsManagementPolicy

### PR DESCRIPTION
**Pull request by Amrita42**
- Applies to main and 4.12
- [Jira](https://issues.redhat.com/browse/CFE-526)
- [Preview](http://file.del.redhat.com/asakthiv/CFE-520-526/networking/ingress-operator.html#nw-configure-dns-mgt-ingress-controller-lb_configuring-ingress)
- @thejasn ptal

Note that Operators API content cant be edited because it is auto-generated.

### CODE REVIEW

**Review by thejasn — CHANGES_REQUESTED** [review]

File: modules/nw-configure-dns-mgt-ingress-controller-lb.adoc [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
++
+[source,terminal]
+----
+oc -n openshift-ingress-operator patch ingresscontrollers/<name> --type=merge --patch='{"spec":{"endpointPublishingStrategy":{"loadBalancer":{"dnsManagementPolicy":"Unmanaged"}}}}'
+----
++
+[NOTE]
+====
+Once you update `.spec.endpointPublishingStrategy.loadBalancer.dnsManagementPolicy` field to `Unmanaged`, the existing DNS record must denote the same using the `dnsManagementPolicy` field. You can either retain or delete the DNS record in the cloud provider.
+If the domain on the Ingress Controller does not match with `dnsConfig.Spec.BaseDomain` , the policy is automatically set to `Unmanaged`.
```
Comment (thejasn) [inline modules/nw-configure-dns-mgt-ingress-controller-lb.adoc:38]: This is only available on AWS cloud.

**Review by thejasn — CHANGES_REQUESTED** [review]

File: modules/nw-configure-dns-mgt-ingress-controller-lb.adoc (hunk 1 above)
Comment (thejasn) [reply modules/nw-configure-dns-mgt-ingress-controller-lb.adoc:38]:
```suggestion
On AWS cloud plaform, if the domain on the Ingress Controller does not match with `dnsConfig.Spec.BaseDomain` , the `dnsManagementPolicy` on the Ingress Controller is set to `Unmanaged`.
```

File: modules/nw-configure-dns-mgt-ingress-controller-lb.adoc [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+. Save the file to apply the changes.
++
+[source,terminal]
+----
+oc apply -f <name>.yaml <1>
+----
+
+. Optional: You can delete the associated DNS record in the cloud provider.
+
```
[the remark this answers is not in the document]
Comment (Amrita42) [reply modules/nw-configure-dns-mgt-ingress-controller-lb.adoc:72]: Its not possible to add a link/xref within the module due to tool limitations however we can add it here >> Additional Resources >>https://docs.openshift.com/container-platform/4.11/networking/ingress-operator.html#additional-resources

File: modules/nw-configure-dns-mgt-ingress-controller-lb.adoc [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Log in as a user with `cluster-admin` privileges.
+* DNS record created.
+
+.Procedure
+
+. Modify an `IngressController` custom resource to set `dnsManagementPolicy`:
```
[the remark this answers is not in the document]
Comment (Amrita42) [reply modules/nw-configure-dns-mgt-ingress-controller-lb.adoc:28]: Yes, I think it is a good idea to document both.

**Comment by Amrita42** [comment]
Closing this PR , opened a new PR >> https://github.com/openshift/openshift-docs/pull/49752

**Pull request closed without merging**